### ISSUE TITLE
test querying last uploaded in Micropub media endpoint

support in wp micropub: https://github.com/indieweb/wordpress-micropub/issues/190

### ISSUE THREAD

**Issue by dougbeal** [aaronpk/micropub.rocks#83]
test querying last uploaded in Micropub media endpoint

support in wp micropub: https://github.com/indieweb/wordpress-micropub/issues/190

**Comment by dougbeal** [comment]
micropub .rocks should request media access as well?

**Comment by jalcine** [comment]
Yeah, I just ran into this myself. My result page is over at https://micropub.rocks/server-tests/104?endpoint=359

(Originally published at: https://v2.jacky.wtf/post/aac23679-d61a-48dd-a91a-75db49cdb21c)
